lib/components/wallet_connect/approve_token_transfer.dart

```diff
@@ -240,7 +258,25 @@ class _ApproveTokenTransferState extends State<ApproveTokenTransfer> {
                             style: TextStyles.lightGreyTextSmall,
                           ),
                     ThemedControls.spacerVerticalMini(),
-                    Text(widget.toID ?? "-", style: TextStyles.textNormal)
+                    Text(widget.toID ?? "-", style: TextStyles.textNormal),
```

Comment:
@ahmed-tarek-salem  fromId and toID can't be null at this point, it should have enough validations before, otherwise the trx will fail. I'm curious why the condition to set - is needed.